ngraph/core/src/validation_util.cpp

```diff
@@ -954,17 +954,18 @@ void ngraph::opset1::infer_conv_backprop_auto_padding(const Shape& input_data_sh
 
     for (uint64_t i = 0; i < num_spatial_dims; ++i)
     {
-        int total_padding = strides[i] * (input_data_shape[i] - 1) +
-                            dilations[i] * (filters_shape[i] - 1) + 1 - output_shape[i] +
-                            output_padding[i];
+        int total_padding = std::max<int>(strides[i] * (input_data_shape[i] - 1) +
+                                              dilations[i] * (filters_shape[i] - 1) + 1 -
+                                              output_shape[i] + output_padding[i],
+                                          0);
         if (auto_pad_type != op::PadType::SAME_UPPER)
         {
-            pads_begin[i] = total_padding / 2;
```

Comment:
@lazarevevgeny I just improved performance a little bit 😄 
[Multiplication is faster than division](https://stackoverflow.com/questions/17883240/is-multiplication-faster-than-float-division/17883577#:~:text=3%20Answers&text=Multiplication%20is%20faster%20than%20division.,even%20as%20slow%20as%20division.)

Reply:
Division of an integer number by 2 is just one bit shift. While multiplication with a floating point number is way more complex since it require dedicated ALU. So I would revert this change.

Reply:
Ok, I will revert these changes.